endpoints/post_websites-scan.md

```diff
@@ -6,13 +6,17 @@ Trigger scan of one or more websites
 [
 	{
 		"account_id": "<string>",
-		"website_id": "<string>"
+		"website_id": "<string>",
+		"scan_type": "<enum{'one_page', 'all_page'}>"
 	}
 ]
 ```
 
 * `account_id` is the unique identifier of the account which owns the website.
 * `website_id` is the unique identifier of website to scan.
+* `scan_type' indicates what type of scan should be run. If no value is provided the default is `all_page`.
```

Comment:
It's really `all_page` instead of `all_pages`?

You *do* know I'm a card-carrying member of the Grammar Gestapo, right? As such, I find this "not in order."
![grammar-gestapo](https://user-images.githubusercontent.com/71285654/142468236-df4c6b05-9b5b-4411-a522-b68260ee3afa.gif)

Reply:
ja